packages/theme-base/src/css/component/button.scss

```diff
@@ -3,6 +3,7 @@
  */
 
 .amplify-button {
+  display: inline-block;
```

Comment:
This is the default browser display for buttons, which was unset to inline by our `all: unset` rule.